sensor/common/networkflow/service/service_impl.go

```diff
@@ -88,16 +89,23 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 	return s.authFuncOverride(ctx, fullMethodName)
 }
 
-// PushNetworkConnectionInfo handles the bidirectional gRPC stream with the collector
-func (s *serviceImpl) PushNetworkConnectionInfo(stream sensor.NetworkConnectionInfoService_PushNetworkConnectionInfoServer) error {
+// Leaving this commented out for now. Will not merge like this
+//// PushNetworkConnectionInfo handles the bidirectional gRPC stream with the collector
+// func (s *serviceImpl) PushNetworkConnectionInfo(stream sensor.NetworkConnectionInfoService_CommunicateServer) error {
```

Comment:
I think we need to be backwards compatible to not break StackRox. We should not break the gRPC connection.